internal/pkg/model/pathstate.go

```diff
@@ -91,6 +101,36 @@ func (f *PathsState) UntrackedPaths() []string {
 	return untracked
 }
 
+func (f *PathsState) UntrackedDirs() (dirs []string) {
+	for _, path := range f.UntrackedPaths() {
+		if !f.fs.IsDir(path) {
+			continue
+		}
+		dirs = append(dirs, path)
+	}
+	return dirs
+}
+
+func (f *PathsState) UntrackedDirsFrom(base string) (dirs []string) {
+	for _, path := range f.UntrackedPaths() {
+		if !f.fs.IsDir(path) || !filesystem.IsFrom(path, base) {
+			continue
+		}
+		dirs = append(dirs, path)
+	}
+	return dirs
+}
+
+func (f *PathsState) LogUntrackedPaths(logger *zap.SugaredLogger) {
+	untracked := f.UntrackedPaths()
+	if len(untracked) > 0 {
+		logger.Warn("Unknown paths found:")
+		for _, path := range untracked {
+			logger.Warn("\t- ", path)
```

Comment:
No, pokud to půjde ven až po https://github.com/keboola/keboola-as-code/pull/215 tak by tady měly být místo tabulátoru ty dvě mezery.

Reply:
Přemýšlím jestli teda nedoplnit i ten logger o nějakou další convenience metodu která by tam to odsazení doplňovala. (Nebo ty jsi vlastně říkal něco o tom že se to logování bude ještě měnit?)